src/MQTT.h

```diff
@@ -76,10 +76,8 @@ bool mqtt_publish(const char *reading, char *payload, boolean retain) {
 /**
  * @brief Publish Data to MQTT
  */
-bool mqtt_publish(const char *reading, char *payload) {
-    char topic[120];
-    snprintf(topic, 120, "%s%s/%s", mqtt_topic_prefix, hostname, reading);
-    return mqtt.publish(topic, payload, false);
+bool mqtt_publish(const char *reading, char const *payload) {
+    return mqtt_publish(reading, payload, false);
```

Comment:
This method doesn't do anything anymore, so can be removed. But is it really correct to remove the topic etc.?

Reply:
Or even just add an overload for mqtt_publish that accepts doubles

Reply:
In 03043fbe4efc34a664eff47dddc81b53f7b502b8 I overloaded this function to allow for explicit setting of the retain behaviour. At the time, I just duplicated the code, but afterwards realized, that it could be unified this way.

(just to be sure: you have noticed, that it is mqtt_publish (underscore) now instead of mqtt.publish (dot)?)

Reply:
Ah ok, I did overlook that it changed from . to _. So topic is fine, but the method is still not really doing anything except hiding the retain parameter.

Reply:
You can also define the same by
`bool mqtt_publish(const char *reading, char const *payload, boolean retain = false) {
`

Reply:
Ok thank you for the advice, I have modified it accordingly.